packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -8,6 +8,13 @@ import { CfnOutput, Construct, Stack } from '@aws-cdk/core';
  * The properties for the base QueueProcessingEc2Service or QueueProcessingFargateService service.
  */
 export interface QueueProcessingServiceBaseProps {
+  /**
+   * The name of the Fargate service.
```

Comment:
This comment is inaccurate, it should be `The name of the service` as this property is used for both an Ec2Service and a FargateService.